scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/package.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.parquet
+
+import java.lang.{Boolean => JBoolean}
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.testing.AvroIO
+import com.spotify.scio.util.ScioUtil
+import com.spotify.scio.values.SCollection
+import org.apache.avro.Schema
+import org.apache.avro.specific.SpecificRecordBase
+import org.apache.beam.sdk.io.hadoop.inputformat.HadoopInputFormatIO
+import org.apache.beam.sdk.transforms.SimpleFunction
+import org.apache.hadoop.mapreduce.Job
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat
+import org.apache.parquet.avro.AvroParquetInputFormat
+import org.apache.parquet.filter2.predicate.FilterPredicate
+import org.apache.parquet.hadoop.ParquetInputFormat
+
+import scala.reflect.ClassTag
+
+/**
+ * Main package for Parquet Avro APIs. Import all.
+ *
+ * {{{
+ * import com.spotify.scio.parquet.avro._
+ * }}}
+ */
+package object avro {
+
+  /** Alias for [[me.lyh.parquet.avro.Projection]]. */
+  val Projection = me.lyh.parquet.avro.Projection
+
+  /** Alias for [[me.lyh.parquet.avro.Predicate]]. */
+  val Predicate = me.lyh.parquet.avro.Predicate
+
+  /** Enhanced version of [[ScioContext]] with Parquet Avro methods. */
+  implicit class ParquetAvroScioContext(val self: ScioContext) extends AnyVal {
+
+    // GenericRecord is not supported since HadoopInputFormatIO cannot infer Coder from type
+    /**
+     * Get an SCollection for a Parquet file as Avro records.
```

Comment:
Not sure where is the best place to put this, but it might be useful to comment/document the limitations of Hadoop*IO, mainly the liquid sharding, thus whatever is gonna be the initial split will stay the same until next fusion.